### PR TITLE
Sign in

### PR DESCRIPTION
**Pull request by HarryPerez**
## Trello Card
https://trello.com/c/GxapVSda/3-sign-in

### CODE REVIEW

File: app/routes.js [hunk 1]
```diff
@@ -1,4 +1,8 @@
 const users = require('./controllers/users.js'),
-  signUpValidator = require('./middlewares/validator').signUpValidator;
+  signUpValidator = require('./middlewares/validator').signUpValidator,
+  signInValidator = require('./middlewares/validator').signInValidator;
 
-exports.init = app => app.post('/users', [signUpValidator], users.create);
+exports.init = app => {
+  app.post('/users', [signUpValidator], users.create),
+    app.post('/users/sessions', [signInValidator], users.signIn);
```
Comment (molinajulian) [inline app/routes.js:7]: This space is weird

File: test/user.js [hunk 2]
```diff
@@ -71,4 +74,68 @@ describe('/users POST', () => {
         res.body.message[0].param.should.equal('firstName');
         res.body.message[0].msg.should.equal('can not be empty');
       }));
+  it('should be successful signing in', () =>
```
Comment (molinajulian) [inline test/user.js]: Move this test for another describe

File: test/user.js [hunk 3]
```diff
@@ -71,4 +74,68 @@ describe('/users POST', () => {
...
+    chai
+      .request(server)
+      .post('/users/sessions')
+      .send({
+        email: 'test@wolox.com.ar',
+        password: '11122233'
+      })
+      .then(res => {
+        res.should.have.status(422);
+        res.body.message.should.equal('The password: 11122233 is not valid');
```
Comment (molinajulian) [inline test/user.js]: This is wrong. The error can't show the password sended.

Comment (molinajulian) [reply test/user.js]: Maybe `The credentials are not valid ` or similar

**Review by molinajulian — CHANGES_REQUESTED** [review]

File: app/routes.js (hunk 1 above)
Comment (HarryPerez) [reply app/routes.js:7]: This space is being applied by the linter of the bootstrap. It is automatic.

Comment (molinajulian) [reply app/routes.js:7]: Yes, maybe. We create our linter in this moment. :+1:

**Review by molinajulian — APPROVED** [review]

**Pull request merged**